components/vf-show-more/vf-show-more.scss

```diff
@@ -0,0 +1,83 @@
+// vf-show-more
+
+// **Thinking about deleting this file?**
+// If your component needs no CSS/Sass, we still recommend leaving the
+// scss files in place. As this is primarily a CSS framework, it is better to
+// leave the empty files so you know a file wasn't accidently omitted.
+// If you don't have any Sass, you can trim this block down to:
+// "This page was intentionally left blank"
+
+@import 'package.variables.scss';
+// Debug information from component's `package.json`:
+// ---
+/*!
+ * Component: #{map-get($componentInfo, 'name')}
+ * Version: #{map-get($componentInfo, 'version')}
+ * Location: #{map-get($componentInfo, 'location')}
+ */
+
+@import 'vf-show-more.variables.scss';
+
+// You need to add this Sass file to ./components/vf-componenet-rollup/index.scss
+// @import 'vf-show-more/vf-show-more.scss';
+
+// Animation
+// Hiding CSS is added in JS
+.vf-show-more .vf-show-more--item {
```

Comment:
I'm not sure we need to increase the specificity of the CSS here? It could just be `.vf-show-more--item` etc.

Similarly with most of this CSS

Reply:
Now done